Add the armv5e architecture to binutils

Binutils has a comment that indicates it is supposed to match gcc for all of
the support "-march=" settings, but it was lacking the armv5e setting.  This
was a simple way to add it, as thumb instructions shouldn't be generated by
the compiler anyway.

Signed-off-by: Mark Hatle <mark.hatle@windriver.com>

diff -urN binutils-2.22.orig/gas/config/tc-arm.c binutils-2.22/gas/config/tc-arm.c
--- binutils-2.22.orig/gas/config/tc-arm.c	2011-11-21 03:29:31.000000000 -0600
+++ binutils-2.22/gas/config/tc-arm.c	2012-04-03 12:07:42.230158760 -0500
@@ -22990,6 +22990,7 @@
   {"armv4t",		ARM_ARCH_V4T,	 FPU_ARCH_FPA},
   {"armv4txm",		ARM_ARCH_V4TxM,	 FPU_ARCH_FPA},
   {"armv5",		ARM_ARCH_V5,	 FPU_ARCH_VFP},
+  {"armv5e",		ARM_ARCH_V5TE,	 FPU_ARCH_VFP},
   {"armv5t",		ARM_ARCH_V5T,	 FPU_ARCH_VFP},
   {"armv5txm",		ARM_ARCH_V5TxM,	 FPU_ARCH_VFP},
   {"armv5te",		ARM_ARCH_V5TE,	 FPU_ARCH_VFP},
